docs/models.txt

```diff
@@ -685,10 +686,10 @@ Interface mediafile {
     mimetype: string;
     pdf_information: JSON;
     create_timestamp: datetime;
-    path: string; // Note: calcuated
-    inherited_access_group_ids: boolean | number[]; // Note: calculated and no
-    // reverse-relation for the group ids.
+    path: string; // Note: calculated
 
+    has_inherited_access_groups: boolean; // Note: calculated
```

Comment:
Ok. false=public, true=Lookup in `inherited_access_group_ids`; if `inherited_access_group_ids` is `[]`, no perms, correct?

Reply:
Yes (except that `access_group_ids` also has to be checked, that's why I've named it so explicit and not more general)

Reply:
> except that access_group_ids also has to be checked

does it? It should be enough to check `*inherited_access_groups` since `access_group_ids` should be evaluated in the inherited ones, too.

Reply:
Ah okay, I thought `inherited_access_group_ids` only included those from above and not `access_group_ids` themselves. But well, the naming is still fine I think